package.json

```diff
@@ -20,7 +20,7 @@
     "create-react-component": "yo react-domain"
   },
   "dependencies": {
-    "@theforeman/vendor": "^3.3.2",
```

Comment:
Need to update all the `@theforeman/*` deps

Reply:
update all to `3.5.2` - the latest

Reply:
https://github.com/theforeman/foreman-packaging/pull/4461